<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetre</class>
 <widget class="QMainWindow" name="Fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenetre</string>
  </property>
  <widget class="QWidget" name="mainFrame">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="modification">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="searchCardButton">
         <property name="text">
          <string>Recherche carte</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="enrollButton">
         <property name="text">
          <string>Enroll</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="formatButton">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="modificationCredit">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="incrementButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decrementButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="informations">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameValue">
         <property name="maxLength">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="firstNameLabel">
         <property name="text">
          <string>Prénom :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="creditLabel">
         <property name="text">
          <string>Crédit : </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="creditValue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="firstNameValue">
         <property name="maxLength">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnexion">
    <property name="title">
     <string>Connexion</string>
    </property>
    <addaction name="actionConnecter_le_lecteur"/>
    <addaction name="actionD_connecter_le_lecteur"/>
   </widget>
   <addaction name="menuConnexion"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnecter_le_lecteur">
   <property name="text">
    <string>Connecter le lecteur</string>
   </property>
  </action>
  <action name="actionD_connecter_le_lecteur">
   <property name="text">
    <string>Déconnecter le lecteur</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
